decred.go

```diff
@@ -170,6 +170,11 @@ func main() {
 
 	c := newCrawler(cfg.netParams, amgr)
 
+	server, err := newServer(cfg.Listen, amgr)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Should follow the pattern used by the rest (`fmt.Fprintf` followed by `os.Exit`).

Reply:
Fixed